lib/pengine/utils.c

```diff
@@ -2215,7 +2215,7 @@ calculate_restart_digest(op_digest_cache_t *data, xmlNode *xml_op,
  *
  * \return Pointer to new digest cache entry (or NULL on memory error)
  * \note It is the caller's responsibility to free the result using
- *       destroy_digest_cache().
+ *       pe__free_digest().
```

Comment:
above it is called pe__free_digests - which is probably the more consistent naming

Reply:
Good catch